common/unsafe/src/test/java/org/apache/spark/unsafe/PlatformUtilSuite.java

```diff
@@ -58,4 +61,17 @@ public void overlappingCopyMemory() {
       Assert.assertEquals((byte)i, data[i + 1]);
     }
   }
+
+  @Test
+  public void memoryDebugFillEnabledInTest() {
+    Assert.assertTrue(MemoryAllocator.MEMORY_DEBUG_FILL_ENABLED);
```

Comment:
This assertion fails in Jenkins.
Did you intend to set the flag to true in a static block at the beginning of the test ?